antarest/study/storage/abstract_storage_service.py

```diff
@@ -337,6 +337,50 @@ def export_study(
             )
         return target
 
+    def _export_output(
+        self, metadata: T, output_id: str, target: Path
+    ) -> Path:
+        path_output = Path(metadata.path) / "output" / output_id
```

Comment:
Add a log "Exporting output [...]"

Reply:
check if the path_output exists, and raise a new exceptions "StudyOutputNotFound" (to add in core/exceptions)